pkgs/applications/graphics/photoflow/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, fetchFromGitHub, gettext, glib, libxml2, pkgconfig, swig, automake, gobjectIntrospection, cmake, libtiff, libjpeg, fftw, exiv2, lensfun, gtkmm2, libraw, lcms2, libexif, vips, expat, pcre, pugixml }:
+
+stdenv.mkDerivation {
+  name = "photoflow-0.2.8";
+
+  src = fetchFromGitHub {
+    owner = "aferrero2707";
+    repo = "PhotoFlow";
+    rev = "914561fcb1ca97de4814c4024d9a76e4b435c2af";
+    sha256 = "06gsw0yhkdnhgcvnbsc5s4agm678ap10djvji8m2qssgnjn26vgs";
+  };
+
+  buildInputs = [ gettext glib libxml2 pkgconfig swig automake gobjectIntrospection cmake libtiff libjpeg fftw exiv2 lensfun gtkmm2 libraw lcms2 libexif vips expat pcre pugixml ];
```

Comment:
Tools used for building the software (`gettext libxml2 pkgconfig swig automake gobjectIntrospection cmake`) should go to `nativeBuildInputs`.

Reply:
Also when using `cmake`, you should also add `ninja`.

Reply:
Also, you should use `gtkmm3` in accordance with https://github.com/NixOS/nixpkgs/issues/18559

Reply:
Photoflow can be compiled with gtkmm3 but unfortunately it's not very usable.
Discussed here: https://discuss.pixls.us/t/help-needed-for-gtk3-theme/5803

Reply:
Maybe mention it in comment, them.

Reply:
That makes sense. Anyway thanks for great review.